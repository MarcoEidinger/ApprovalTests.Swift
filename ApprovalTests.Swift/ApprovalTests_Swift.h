//
//  ApprovalTests_Swift.h
//  ApprovalTests.Swift
//
//  Created by Industrial Logic on 9/29/18.
//  Copyright Â© 2018 NA. All rights reserved.
//

#import <Cocoa/Cocoa.h>

//! Project version number for ApprovalTests_Swift.
FOUNDATION_EXPORT double ApprovalTests_SwiftVersionNumber;

//! Project version string for ApprovalTests_Swift.
FOUNDATION_EXPORT const unsigned char ApprovalTests_SwiftVersionString[];

// In this header, you should import all the public headers of your framework using statements like #import <ApprovalTests_Swift/PublicHeader.h>


