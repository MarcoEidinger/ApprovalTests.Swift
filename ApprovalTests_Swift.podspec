#
#  Be sure to run `pod spec lint ApprovalTests_Swift.podspec' to ensure this is a
#  valid spec and to remove all comments including this before submitting the spec.
#
#  To learn more about Podspec attributes see http://docs.cocoapods.org/specification.html
#  To see working Podspecs in the CocoaPods repo see https://github.com/CocoaPods/Specs/
#

Pod::Spec.new do |s|
  s.name         = "ApprovalTests_Swift"
  s.version      = "0.0.1"
  s.summary      = "ApprovalTests for Swift."

  s.description  = <<-DESC
ApprovalTests for Swift
                   DESC

  s.homepage     = "https://github.com/approvals"

  s.license      = "Apache 2.0 License"
  # s.license      = { :type => "MIT", :file => "FILE_LICENSE" }

  s.author    = "Matthew Carlson"

  s.platform = :osx
  s.osx.deployment_target = "10.13"

  s.source       = { :git => "https://github.com/approvals/ApprovalTests.Swift.git" }


  s.source_files  = "ApprovalTests.Swift", "ApprovalTests.Swift/**/*.swift"
  #s.exclude_files = "Classes/Exclude"

  # s.public_header_files = "Classes/**/*.h"
end
