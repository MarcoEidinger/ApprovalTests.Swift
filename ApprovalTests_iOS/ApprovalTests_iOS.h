#import <Foundation/Foundation.h>

FOUNDATION_EXPORT double ApprovalTests_iOSVersionNumber;
FOUNDATION_EXPORT const unsigned char ApprovalTests_iOSVersionString[];
