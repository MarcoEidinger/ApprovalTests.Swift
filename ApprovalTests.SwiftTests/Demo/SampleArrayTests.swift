#if os(OSX)
    import ApprovalTests_Swift
#elseif os(iOS)
    import ApprovalTests_iOS
#endif
import XCTest

final class SampleArrayTests: XCTestCase {

    func testList() throws {
        var names = ["Llewellyn", "James", "Dan", "Jason", "Katrina"]
        names.sort()
        try Approvals.verifyAll(label: "", names)
    }
}
