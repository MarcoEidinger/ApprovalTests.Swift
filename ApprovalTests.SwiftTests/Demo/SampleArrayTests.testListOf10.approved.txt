Greek[00] = alpha
Greek[01] = beta
Greek[02] = gamma
Greek[03] = delta
Greek[04] = epsilon
Greek[05] = zeta 
Greek[06] = eta
Greek[07] = theta
Greek[08] = iota
Greek[09] = kappa
